make_prg/__main__.py

```diff
@@ -17,23 +19,28 @@ def main():
         title="Available subcommands", help="", metavar=""
     )
 
-    parser.add_argument(
-        "-v",
-        "--verbose",
-        dest="verbose",
-        action="store_true",
-        help="Run with high verbosity (debug level logging)",
-    )
+    msa_parser = from_msa.register_parser(subparsers)
+    update_parser = update.register_parser(subparsers)
 
-    from_msa.register_parser(subparsers)
+    for par in [msa_parser, update_parser]:
+        par.add_argument(
+            "-v",
+            "--verbose",
+            action='count',
+            default=0,
+            help="Increase output verbosity (-v for debug, -vv for trace - trace is for developers only)",
+        )
+        par.add_argument("--log", help="Path to write log to. Default is stderr")
 
     args = parser.parse_args()
 
-    if args.verbose:
-        log_level = logging.DEBUG
-    else:
-        log_level = logging.INFO
-    logging.basicConfig(level=log_level, handlers=[])
+    log_levels = ["INFO", "DEBUG", "TRACE"]
+    log_level = log_levels[min(args.verbose, 2)]
```

Comment:
```suggestion
    log_level = log_levels[min(args.verbose, len(log_levels) - 1)]
```
Slightly less error-prone and no need to remember to change the 2 if we add more log levels

Reply:
Agreed